scripts/build/rollup.config.js

```diff
@@ -54,6 +55,39 @@ export const umd = {
   ],
 };
 
+const updateLibPrefix = (isUndo) => {
```

Comment:
Nit: Should function name give a bit more context? Not sure if I'm just unfamiliar with rollup/context of the PR. `isUndo` makes a bit more sense with some context.